plugin/omero_iviewer/views.py

```diff
@@ -188,3 +208,87 @@ def request_rois(request, iid, conn=None, **kwargs):
         return JsonResponse(ret, safe=False)
     except Exception as someException:
         return JsonResponse({"error": repr(someException)})
+
+
+@login_required()
+def image_data(request, image_id, conn=None, **kwargs):
+    if image_id is None:
```

Comment:
Using the regex suggested above in urls.py, it shouldn't be necessary to check if None here.